app/src/main/java/org/mifos/mobilebanking/presenters/LoginPresenter.java

```diff
@@ -195,9 +195,12 @@ private boolean isCredentialsValid(final String username, final String password)
                     , resources.getString(R.string.password), resources.getInteger(R.integer.
                             password_minimum_length)));
             return false;
+        } else {
```

Comment:
@Rish02Sharma Its wrong fix, if we do this we will always enter `else` and never call to server will be made.